### DISCUSSION TITLE
Genesenenzertifikat im PU

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#116] [Q&A]
Hallo,

die Anfrage nach einem Genesenen Zertifikat im PU führt bei uns zu einem Fehler.
... https://api.impfnachweis.info:443  ... Executing request POST /api/certify/v2/issue HTTP/1.1 ...

**Sorry, there is something invalid in your request. Error Validating with Schema #: required key [v] not found[\n]**

Die Anfrage-Daten sind:

"{"ver":"1.0.0","nam":{"fn":"Test","gn":"Testa"},"dob":"1990-04-01","r":[{"tg":"840539006","fr":"2021-06-17","df":"2021-06-17","du":"2021-12-14","id":"935339391"}]}"

Das Ganze funktioniert bei uns in der RU Testumgebung und erhalten da ein pdf. Haben wir hier etwas falsch bedacht?

Vielen Dank

**Comment by ghost** [comment]
Können Sie das in der RU erneut probieren? Aktuell bekomme ich dort den gleichen Fehler. Außerdem werden noch Schema-Fehler angemahnt, wenn `co`, `ci` oder `is` im `r`-Element nicht gesetzt sind.

Außerdem: welchen Zweck hat `ver`? In der OpenAPI-spec ist dieses Feld nicht definiert.

**Reply by ghost** [reply]
In der RU funktioniert es nun auch bei mir.

**Reply by marcusthierfelder** [reply]
Vielen Dank, was aber nicht das Problem, schien an der API Version. Bei uns kommt jetzt leider  "400  Sorry, there is something invalid in your request: Test and recoveries certificates are not allowed." ... geht es wirklich bei Ihnren?

**Reply by ghost** [reply]
In der RU ja, PU siehe #66

**Comment by mge-medisoftware** [comment]
In PU wohl ab verfügbar ab 21.6. mit Version API 2.1.0
Siehe https://github.com/Digitaler-Impfnachweis/certification-apis/discussions/66

**Reply by marcusthierfelder** [reply]
Vielen Dank für die Info.
